docs/source/conf.py

```diff
@@ -18,7 +18,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'mlfinlab'
-copyright = '2019, Hudson & Thames,'
+copyright = '2020, Hudson & Thames,'
```

Comment:
Dont update the copyright. The older it is the better. If you update it to 2020 it means the 2019 work was not under CR.